### ISSUE TITLE
Some hints for the Rune words

### ISSUE THREAD

**Issue by SmartArt09** [AlberTajuelo/bitcoin-0.2-image-puzzle#5]
Here's what I could find from all the 2048 bip39 words for the runes:

Rune 1: address (I don't think 'vault' would be the correct word in this context, but I couldn't find any other word anyways)

Rune 2: add/math/total (pretty sure of the first based on the rune translation)

Rune 3: second/day/february/holiday/march/month/october/today (probably the first one as per the order of days in a week but there were some other hints related to dates in the bitcointalk.com discussion that I forgot)

Rune 4: eight/one/seven/six/ten/three/twelve/twenty/two/zero (who knows, maybe some other hint will help here)

**Comment by SmartArt09** [comment]
I also saw no one talking about the number near the statue (1865-202...?)
The only thing I could get from searching is that:

[quote author=Google]The [b]Battle of Fort Myers[/b] was fought on [u]February 20, 1865[/u] (1865-202 --> 20.2.1865), in Lee County, Florida during the last months of the American Civil War.
[/quote]

and 

[quote author=Google]1865 was a pivotal year in American history. With [b]the end of the Civil War, Lincoln's assassination, and the beginning of Reconstruction[/b], politics were forever changed.
[/quote]

These could also be a hint but I have no idea what to link it with.

**Comment by knighthcm** [comment]
Thank for your idea. The problem that I had is some words guess from picture do not have in BIP39. Because English is not my native language so I use Chat GPT to find some synonym words. But still problem that all that synonym words do not have in BIP39 too. For example such as: "lace" guess from picture of "George Floyd"

**Comment by SmartArt09** [comment]
Yeah, no, those kinds of words won't be there in Bip39 English wordlist.
I think the words are to be used as it is if found. There aren't any synonyms that can be used.

Though, if you still want to try that approach, you can take words from Bip39 English wordlist, find their synonyms and see if anything relevant is there in the image.

**Comment by knighthcm** [comment]
Yes, thank you for sharing your think. Do you have any idea on number 6

**Comment by SmartArt09** [comment]
Unfortunately, I don't have any idea of the order of the words

**Comment by Rscarp79** [comment]
Whay about the hidden word on the space needle and on the bloody neck

**Comment by Rscarp79** [comment]
I used photoshop but don't know how to use properly. And I think there is instructions on the welcome to ...

**Comment by SmartArt09** [comment]
> Whay about the hidden word on the space needle and on the bloody neck

That's BREATHE, but it is probably a passphrase as it is not in the list of BIP39 English words.

**Comment by knighthcm** [comment]
> Whay about the hidden word on the space needle and on the bloody neck

on the space needle may be it says about seedphrase related to food

**Comment by knighthcm** [comment]
> I used photoshop but don't know how to use properly. And I think there is instructions on the welcome to ...

I will try

**Comment by ExodusHk** [comment]
> Here's what I could find from all the 2048 bip39 words for the runes:
> 
> Rune 1: address (I don't think 'vault' would be the correct word in this context, but I couldn't find any other word anyways)
> 
> Rune 2: add/math/total (pretty sure of the first based on the rune translation)
> 
> Rune 3: second/day/february/holiday/march/month/october/today (probably the first one as per the order of days in a week but there were some other hints related to dates in the bitcointalk.com discussion that I forgot)
> 
> Rune 4: eight/one/seven/six/ten/three/twelve/twenty/two/zero (who knows, maybe some other hint will help here)

i found something interesting contact me beinsports 963@gmail.com

**Comment by knighthcm** [comment]
> > > I used photoshop but don't know how to use properly. And I think there is instructions on the welcome to ...
> > 
> > 
> > I will try
> 
> Good luck, let us know if you find something.

I think some thing such as wrong spell at bitcoin text. Do anyone have idea to find number 12 ? Not much hint for number 12

**Comment by SmartArt09** [comment]
What was number 12?

**Comment by knighthcm** [comment]
Hi SmartArt
if we turn over the picture, so at Trump , the VS will look like 12, but I think it will more difficult, the below there is two line, it direct to the liberty statue, so I think number 12 is concerned liberty but not sure, there is no more double check
